modules/component-store/testing/src/testing.ts

```diff
@@ -0,0 +1,211 @@
+import { Provider, Type } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import 'jasmine';
```

Comment:
Given the hard dep on jasmine here, you'd need to be using Jasmine to use this (which makes sense, given the contribution here).

Does it make sense to put this in

`~/modules/component-store/testing/jasmine/xxx/yyy`

Imaging that the namespace is..

`@ngrx/component-store/testing/jasmine`

Reply:
I wonder if it makes sense to make this generic; the consumer would pass in a parameter that can create a "spy", and a second one that takes a spy and configures it to return a value.

Reply:
Or use a testing framework-agnostic test double library such as [Sinon.JS](https://sinonjs.org/).